Makefile

```diff
@@ -1,39 +1,36 @@
 OR_TOOLS_TOP=../or-tools
-OR_TOOLS_SOURCES=$(OR_TOOLS_TOP)/ortools
 
 TUTORIAL=resources
 
-
-include $(OR_TOOLS_TOP)/Makefile
-CFLAGS := $(CFLAGS) -DOR_TOOLS_VERSION=\"$(OR_TOOLS_VERSION)\"
+CXXFLAGS := -I $(OR_TOOLS_TOP)/include
 
 # For debugging uncomment the next line. -isystem prevents warnings rooted in or-tools library appearing in our compilation
-# CFLAGS := $(CFLAGS) -ggdb -Og -DDEBUG -fsanitize=address -Wall -Wextra -Wshadow -Wunreachable-code -Winit-self -Wmissing-include-dirs -Wswitch-enum -Wfloat-equal -Wundef -isystem$(OR_TOOLS_TOP)/. -isystem$(OR_TOOLS_SOURCES)/gen -isystem$(OR_TOOLS_TOP)/dependencies/install/include -isystem$(OR_TOOLS_TOP)/dependencies/install/include/coin -DUSE_CBC -DUSE_CLP -DUSE_GLOP -DUSE_BOP
+# CFLAGS := $(CFLAGS) -ggdb -Og -DDEBUG -fsanitize=address -Wall -Wextra -Wshadow -Wunreachable-code -Winit-self -Wmissing-include-dirs -Wswitch-enum -Wfloat-equal -Wundef -isystem$(OR_TOOLS_TOP)/. -DUSE_CBC -DUSE_CLP -DUSE_GLOP -DUSE_BOP
```

Comment:
Forgot to rename CFLAGS -> CXXFLAGS

After renaming can you try to compile activating this line to see if it works as intended (i.e., it doesn't show warnings originated form or-tools library)?